internal/controllers/certmanager.go

```diff
@@ -122,6 +122,48 @@ func (r *Reconciler) setupTLS(ctx context.Context, cr *model.CryostatInstance) (
 		return nil, err
 	}
 
+	secret, err := r.GetCertificateSecret(ctx, caCert)
+	if err != nil {
+		return nil, err
+	}
+	// Copy Cryostat CA secret in each target namespace
+	for _, ns := range cr.TargetNamespaces {
+		if ns != cr.InstallNamespace {
+			namespaceSecret := &corev1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      caCert.Spec.SecretName,
+					Namespace: ns,
+				},
+				Type: secret.Type,
```

Comment:
I tried this out in OpenShift and hit the following error:
```
2023-11-29T18:15:34Z ERROR controllers.ClusterCryostat Failed to set up TLS for Cryostat {"Request.Namespace": "cryostat-operator-system", "Request.Name": "clustercryostat-sample", "error": "Secret \"clustercryostat-sample-ca\" is invalid: data[tls.key]: Required value"}
```

I think this is because the original secret uses the type [`kubernetes.io/tls`](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets). Apparently with that type of secret, both the certificate and key are mandatory. We can fix this by changing this to:
```suggestion
				Type: corev1.SecretTypeOpaque,
```

Opaque secrets can contain any arbitrary data.